src/main/java/com/aws/greengrass/easysetup/DeviceProvisioningHelper.java

```diff
@@ -273,17 +274,19 @@ public void cleanThing(IotClient client, ThingInfo thing, boolean deletePolicies
      *
      * To support HTTPS proxies and other custom truststore configurations, append to the file if it exists.
      */
-    private void downloadRootCAToFile(File f) {
+    private void downloadCAToFile(File f, String... urls) {
```

Comment:
why the name change? are CA3 certs not root CA certs?

Reply:
Updated